tensorflow/python/keras/saving/hdf5_format_test.py

```diff
@@ -730,6 +730,48 @@ def test_model_saving_to_pre_created_h5py_file(self):
       os.close(fd)
       os.remove(fname)
 
+  def test_model_saving_to_new_dir_path(self):
+    saved_model_dir = self._save_model_dir()
+    saved_model_dir = os.path.join(saved_model_dir, 'newdir')
+    saved_model_dir = os.path.join(saved_model_dir, 'saved_model')
+    save_format = testing_utils.get_save_format()
+
+    with self.cached_session():
+      model = keras.models.Sequential()
+      model.add(keras.layers.Dense(2, input_shape=(3,)))
+      model.add(keras.layers.RepeatVector(3))
+      model.add(keras.layers.TimeDistributed(keras.layers.Dense(3)))
+
+      x = np.random.random((1, 3))
+      out = model.predict(x)
+
+      keras.models.save_model(model, saved_model_dir, save_format=save_format)
+
+      new_model = keras.models.load_model(saved_model_dir)
+      self._assert_same_weights_and_metrics(model, new_model)
+
+      out2 = new_model.predict(x)
+      self.assertAllClose(out, out2, atol=1e-05)
```

Comment:
any reason to have atol here? I think the out and out2 should be identical.

Reply:
The existing tests have this atol. I assumed it is there in case there is any inaccuracy when saving/loading?  So I just did the same thing as the others.